templates/discussions-dashboard.php

```diff
@@ -173,7 +173,19 @@ function( $a, $b ) use ( $latest_comment_date_by_post_id ) {
 			<?php
 		}
 		?>
-		
+		<?php
+			$current_page = max( 1, get_query_var( 'page' ) );
+			echo paginate_links(
+				array(
+					'base'      => add_query_arg( 'page', '%#%' ),
+					'format'    => '?page=%#%',
+					'current'   => $current_page,
+					'total'     => count( $comments ),
```

Comment:
According http://developer.wordpress.org/reference/functions/paginate_links/ this should be the total number of pages.